src/ngrx/3-ngrx-init/ngrx-init.md

```diff
@@ -5,6 +5,44 @@
 
 @body
 
+## NgRx Dependencies
+
+Since we are going to use multiple NgRx tools, let’s install everything we need:
+
+First, run `ng add @ngrx/schematics` to install NgRx Schematics, a tool that will enable us to quickly generate code.
+
+Then, let’s install NgRx dependencies: `npm install @ngrx/{store,effects,entity,store-devtools}@14 --save.`
+
+
+```bash
+ng add @ngrx/schematics@14
+npm install @ngrx/{store,effects,entity,store-devtools}@14 --save
+```
+
+## Login State
+The Login State in our application will be responsible for holding information about authentication and the authenticated user.
+
+### Store Setup
+We’ll take advantage of NgRx Schematics to quickly generate an Authentication Feature State: `run ng generate feature store/auth/Auth -m app.module.ts`.
```

Comment:
This has changed a bit: https://bitovi.atlassian.net/wiki/spaces/NG/pages/210764020/ngrx-chat+branches

When I tried the previously mentioned steps, I couldn't get the project to run ):

Reply:
Do either you or Fabio want to rewrite these instructions so they are accurate? Or do we want me to take a stab at it and then have further review?

Reply:
yup I can update these instructions so this is resolved for this PR